tests/test_app.py

```diff
@@ -0,0 +1,49 @@
+from unittest import mock
+
+import jwt
+import pytest
+
+from gidgethub import app
+
+from .test_abc import MockGitHubAPI
+
+
+class TestGitHubAppUtils:
+
+    """Tests for GitHub App utilities."""
+
+    @mock.patch("time.time")
+    def test_get_jwt(self, time_mock):
+        app_id = 12345
+
+        time_mock.return_value = 1587069751.5588422
+
+        # test file copied from https://github.com/jpadilla/pyjwt/blob/master/tests/keys/testkey_rsa
+        with open("tests/samples/test_rsa_key", "r") as fp:
```

Comment:
Would it be better to use `importlib-resources` or at least our own fixture to read this data? Maybe the code in https://github.com/brettcannon/gidgethub/blob/master/tests/test_sansio.py#L266 should get put into `conftest.py` at minimum?

Reply:
I've changed to using `importlib-resources`.